### ISSUE TITLE
Restrict user's access to Wazuh app

### ISSUE THREAD

**Issue by BASBK** [wazuh/wazuh-kibana-app#2035]
I've install the current version (3.11.1) according to the official documentation. Is it possible to hide some management features (Dev tools, Agents, Management tabs) from specific users? All manuals are refference to older versions of the app and didn't work for me.

**Comment by juankaromo** [comment]
Hi @BASBK 

Unfortunately we cannot hide Tabs by using roles or permissions. It is true that we are working on providing the app with a RBAC system, which will ultimately allow us to perform a set of actions to users depending on their role and permissions. This system will be implemented in Wazuh 4.0, and we have a development issue, if you want you can follow the progress here https://github.com/wazuh/wazuh-kibana-app/issues/619 and here https://github.com/wazuh/wazuh-kibana-app/issues/2021

Despite all this, currently, if we have an Admin mode within the app. What it allows us when deactivating it is that the user can only read the configurations, that is to say, it will not be able to modify nor to create a new configuration. In addition, the set of endpoints of the dev-tools is limited.

It is activated by default, but you can deactivate it from the app's configuration `wazuh.yml`. You have to search for this snippet:

```
# ------------------------------- App privileges --------------------------------
#admin: true
```

And set it to false: 

```
admin: true
```

You will need to restart the app for the changes to take effect.

I hope this helps and please let us know the results.

Regards,

**Comment by juankaromo** [comment]
I close this issue for inactivity. Any other problems or questions please feel free to open a new issue.

Thank you very much for the feedback.

Regards,
